backend/model/model.go

```diff
@@ -1522,6 +1523,37 @@ func MigrateDB(ctx context.Context, DB *gorm.DB) (bool, error) {
 		return false, e.Wrap(err, "Error setting session id sequence to 30000000")
 	}
 
+	if err := DB.Exec(`
+		CREATE TABLE IF NOT EXISTS error_object_embeddings_partitioned
+		(LIKE error_object_embeddings INCLUDING DEFAULTS INCLUDING IDENTITY)
+		PARTITION BY RANGE (project_id);
```

Comment:
I think `PARTITION BY LIST` makes more sense here since we'd always want each partition to have a single project id

Reply:
updated to be `PARTITION BY LIST (project_id);`